File: php/m10.function.php
#!/usr/bin/php
<?php
// create function
function capitalize($string) {
	return strtoupper($string); // return capitlized string
}

// call the function
$result = capitalize("ibm");    // pass string 

# output results with resulting integer
echo "The result of capitalization is: $result.\n";
?>

File: python/m10.function.py
#!/usr/bin/env python
import string
# create the function
def capitalize (string):
   return string.upper() # return fully uppercase string

# output string before calling function
string = "ibm"
print "The current string is: \"%s\"." % string

# call the function
result = capitalize(string)

# output results after calling function
print "The capitalized string is: \"%s\"." % result